build/tools/cloudcore.service

```diff
@@ -3,7 +3,7 @@ Description=cloudcore.service
 
 [Service]
 Type=simple
-ExecStart=/etc/kubeedge/cloudcore
+ExecStart=/usr/local/bin/kubeedge/cloudcore
```

Comment:
In the packages you want to use `/usr/bin` instead of `/usr/local/bin` (the local folder should only be used for programms which are build on the local system

Reply:
@subpathdev Okies. I would make the changes.

Reply:
Done